src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,47 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useContext } from 'react';
+import { ErrorContext } from '../contsxts/ErrorContext';
```

Comment:
The import path for `ErrorContext` is incorrect. It should be '../contexts/ErrorContext' instead of '../contsxts/ErrorContext'.